xml/System.IO/File.xml

```diff
@@ -195,10 +217,30 @@
         <Parameter Name="bytes" Type="System.ReadOnlySpan&lt;System.Byte&gt;" Index="1" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <param name="path">To be added.</param>
-        <param name="bytes">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="path">The file to append to.</param>
+        <param name="bytes">The bytes to append to the file.</param>
+        <summary>
+          <para>Appends the specified byte array to the end of the file at the given path.</para>
+          <para>If the file doesn't exist, this method creates a new file.</para>
+        </summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="path" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentException">
+          <paramref name="path" /> is empty.</exception>
+        <exception cref="T:System.IO.PathTooLongException">The specified path, file name, or both exceed the system-defined maximum length.</exception>
+        <exception cref="T:System.IO.DirectoryNotFoundException">The specified path is invalid (for example, it is on an unmapped drive).</exception>
+        <exception cref="T:System.IO.IOException">An I/O error occurred while opening the file.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">
+          <paramref name="path" /> specified a file that is read-only.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">
+          <paramref name="path" /> specified a file that is hidden.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">
+          <paramref name="path" /> specified a directory.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">The caller does not have the required permission.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">This operation is not supported on the current platform.</exception>
```

Comment:
As @Jozkee pointed out, these should be formatted like so:

```suggestion
        <exception cref="T:System.UnauthorizedAccessException">
          <paramref name="path" /> specified a file that is read-only.

          -or-

          <paramref name="path" /> specified a file that is hidden.

          -or-
          
          <paramref name="path" /> specified a directory.

          -or-

          The caller does not have the required permission.

          -or-

          This operation is not supported on the current platform.</exception>
```

Reply:
> Also, quick Q, is the last one correct? I would think that would be a PlatformNotSupportedException?

@adegeo it was most likely just taken from an already documented method that is similar to this one e.g. https://learn.microsoft.com/en-us/dotnet/api/system.io.file.writealltext?view=net-8.0.